src/github/pull-request.ts

```diff
@@ -25,12 +26,11 @@ export const pullRequestWebhookHandler = async (context: CustomContext, jiraClie
 	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, jiraClient.baseURL, context.log);
 
 	context.log = context.log.child({
-		jiraHostName: jiraClient.baseURL,
+		jiraHost: createHashWithSharedSecret(jiraClient.baseURL),
```

Comment:
changed to just jiraHost because jiraHostName is a lie.
Plus it is the only time we refer to it as jiraHostName.

Reply:
wouldn't it be simpler to just add a top level serializer in `logger.ts` to check for all keys that should be encrypted and do it there instead of duplicating this logic everywhere?

Reply:
I agree with @mboudreau ! But did tried couple way and could not figure out how to do this. Try to reference to [this](https://probot.github.io/docs/logging/) and [this](https://getpino.io/#/docs/redaction). 

---

And btw, just figure out that, the probot is using pino log, but in the code, we are typing it as buyan. Although most of the interface are same.

Reply:
Yeah, makes sense, I'll spend some time digging around today.

@gxueatlassian Cheers for the Pino find!!
We are slowly ripping out Probot, so might be a case of top level serializer for our bunyan logge  and do it this way for Pino(depending on effort) for now.

Reply:
Made a serializer, see logger.ts,
**Fun fact**, the values added in the .child function don't go through the serializers :(
so its there for all the log function e.g. `logger.info({**serialize me**}, "some message");`

There is a story after the unsafe data logging to standardize all our logs, so I will leave it until that story to replace .child with create loggers (if appropriate) or move where values are being passed into the logger.